src/core/numerics/ohm/ohm.h

```diff
@@ -321,9 +321,9 @@ namespace core
             for (auto ix = ix0; ix <= ix1; ++ix)
             {
                 Ey(ix) = ideal1D_(Ve, B, {ix}, ComponentTag<Component::Y>{})
-                         + pressure_(n, Pe, {ix}, ComponentTag<Component::Y>{})
-                         + resistive_(J, {ix}, ComponentTag<Component::Y>{})
-                         + hyperresistive_(J, {ix}, ComponentTag<Component::Y>{});
+                         + 0 * pressure_(n, Pe, {ix}, ComponentTag<Component::Y>{})
```

Comment:
please explain

Reply:
the ohm's law is : 

E = -V_e x B - grad(P_e)/(e*n_e) + eta*j - nu*laplacian(J).

-V_e x B is called the "ideal term"
-grad(Pe)/(e*ne) is the "pressure term"
eta*j is the resistive term
-n*laplacian(J) is the hyper resistive term.

as a first step to see whether PHARE works, it's simpler to have only the ideal term in ohm's law, thus zeroing the others.

They will reappear step by step in the (very) near future, but they are tougher to handle.

Reply:
sorry I should have been more specific, I was referring to the "0 *"

Reply:
I know you did, but I gave you a bonus explanation

Reply:
> They will reappear step by step in the (very) near future, but they are tougher to handle.

ok this is the important part I think, the "0 *" is temporary